tests/schemas/test_property_schema.py

```diff
@@ -15,7 +15,7 @@ def test_valid_payload(self, create_property_manager):
             "name": "the heights",
             "address": "111 SW Harrison",
             "city": "Portland",
-            "unit": "101",
+            "num_units": 101,
```

Comment:
This works, though it would probably be cleaner to change this test to use the `property_attributes` fixture instead of a hand-typed dict as the `valid_payload`:

```python
def test_valid_payload(self, property_attributes):
    valid_payload = property_attributes()
    no_validation_errors = {}
    assert no_validation_errors == PropertySchema().validate(valid_payload)
```

If you don't mind, I think it's worth making that change (and deleting the `utils.time` import) since we're already altering the file in this PR.